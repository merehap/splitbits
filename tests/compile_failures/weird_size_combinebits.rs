use splitbits::*;

fn main() {
    let a: u8 = 0b0101_0101;
    combinebits!("aaa aaaa");
}
