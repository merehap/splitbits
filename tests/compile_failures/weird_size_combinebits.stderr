error: proc macro panicked
 --> tests/compile_failures/weird_size_combinebits.rs:5:5
  |
5 |     combinebits!("aaa aaaa");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Template width must be 8, 16, 32, 64, or 128, but was 7.
